hls4ml/backends/vivado/vivado_backend.py

```diff
@@ -8,7 +8,7 @@
 
 from hls4ml.model.types import FixedPrecisionType, NamedType, IntegerPrecisionType
 from hls4ml.model.layers import Layer, Dense, BatchNormalization, Embedding, Conv1D, Conv2D, Conv2DBatchnorm, SeparableConv1D, SeparableConv2D, DepthwiseConv2D, Activation, ParametrizedActivation, PReLU, Softmax, Pooling1D, Pooling2D, GlobalPooling1D, GlobalPooling2D, ZeroPadding1D, ZeroPadding2D, Merge, Concatenate, Dot, Resize, Transpose, SimpleRNN, LSTM, GRU, GarNet, GarNetStack
```

Comment:
Same comment here about avoiding super-long lines with parentheses, like https://github.com/matplotlib/matplotlib/blob/main/lib/matplotlib/backends/backend_qt.py#L9-L12.

Reply:
`pre-commit` will handle this